### ISSUE TITLE
This plugin is incompatible with asdf-java

### ISSUE THREAD

**Issue by dklotz** [vic/asdf-graalvm#4]
Currently this plugin is not really usable alongside [asdf-java](https://github.com/halcyon/asdf-java), since both want to install shims for `java`, `javac` etc. **Edit:** To be precise, when I have both a jdk (through the java plugin) and graalvm 19.0.0 installed, `java` etc. still point to the versions from the jdk while graalvm-specific binaries like `gu` point to graalvm. And I don't really know how I would solve this.

~~I couldn't find any general asdf "policy" for different plugins that use the same binary names, don't know if this ever came up?~~

~~One idea for compatibility might be to e.g. deprecate this plugin and add support for graalvm-x.y.z "versions" to the java plugin instead?~~

**Comment by dklotz** [comment]
Ok, maybe I was a bit too fast there: I now found [this closed issue](https://github.com/asdf-vm/asdf/issues/431) and [this PR](https://github.com/asdf-vm/asdf/pull/432). I experimented a bit more, and it seems to work as long as I don't have a global version of Java set in my `~/.tool-versions`. As long as my individual project folders then have either a `java` or a `graalvm` version set in their `.tool-versions`, `java` etc. will point to the right version. As soon as I have a global default java version, that always seems to override the tools from graalvm, even if I am in a folder that just has `graalvm` in the `.tool-versions`.
